src/main/java/net/minecraftforge/client/extensions/IForgeEntityModel.java

```diff
@@ -0,0 +1,31 @@
+package net.minecraftforge.client.extensions;
+
+import net.minecraft.client.model.geom.ModelPart;
+import net.minecraft.world.entity.Entity;
+import net.minecraftforge.client.model.animation.IEntityModelAnimationModifier;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public interface IForgeEntityModel<T extends Entity>
+{
+    Set<IEntityModelAnimationModifier> animationModifiers = new HashSet<>();
```

Comment:
I believe this won't work as you expect, as all fields in an interface are implicitly `public static final`, so all of these animation modifiers will be shared across all `IForgeEntityModel` instances.